release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3736,4 +3736,30 @@ $ oc adm release info 4.12.20 --pullspecs
 [id="ocp-4-12-20-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-21"]
+=== RHBA-2023:3546 - {product-title} 4.12.21 bug fix update and security update
+
+Issued: 2023-06-14
+
+{product-title} release 4.12.21, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:3546[RHBA-2023:3546] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3545[RHSA-2023:3545] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.21 --pullspecs
+----
+
+[id="ocp-4-12-21-bug-fixes"]
+==== Bug fixes
+
+* Previously on {sno}, in case of node reboot there was a race condition that could result in admission of application pods requesting devices on the node even if devices were unhealthy or unavailable to be allocate. This resulted in runtime failures when the application tried to access devices. With this update, the resources requested by the pod are only allocated if the device plugin has registered itself to Kubelet and healthy devices are present on the node to be allocated.
```

Comment:
```suggestion
* Previously on {sno}, in case of node reboot there was a race condition that could result in the admission of application pods requesting devices on the node even if devices were unhealthy or unavailable to be allocated. This resulted in runtime failures when the application tried to access devices. With this update, the resources requested by the pod are only allocated if the device plugin has registered itself to kubelet and healthy devices are present on the node to be allocated.
```

Reply:
Changes made.